index.js

```diff
@@ -12,7 +12,8 @@ try {
  * Element prototype.
  */
 
-var proto = Element.prototype;
+var Element = require('global-object').Element;
```

Comment:
I think it's better to use another dep then to copy and paste the code into this library...